### PR TITLE
Set map format to sidemap

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Set the map format to sidemap, to better represent that we're in 3/4th perspective now.
Some code cleanup around the definition of /world.

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

File: code/hub.dm [hunk 1]
```diff
@@ -1,16 +0,0 @@
-#define HUB_ENABLED
-
-/world
-	hub = "Exadv1.spacestation13"
-	name = "Space Station 13"
-#ifdef HUB_ENABLED
```
Comment (Arrow768) [inline code/hub.dm:6]: This needs to be maintained in the new version.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/hub.dm (hunk 1 above)
Comment (FluffyGhoster) [reply code/hub.dm:6]: For the reason of...? It's literally defined at the top of the file, doing nothing

Comment (Arrow768) [reply code/hub.dm:6]: Yeah, you are right, I remembered the old system where the hub password was not set by default

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**